doc/dev-guide.rst

```diff
@@ -35,7 +35,7 @@ work, you can just source the virtual environment directly:
 
 .. code-block:: bash
 
-  source .venv/bin/activate
+  user@host:~/katgpucbf$ source .venv/bin/activate
```

Comment:
I don't really think this is necessary. The code block was sufficient.

Reply:
I included it here (and below) to more clearly imply/show without explicitly saying where the commands should be run from (i.e. root katgpucbf directory).

Reply:
Too right, too right. Will roll back and update accordingly.